src/components/AppLayout/index.tsx

```diff
@@ -1,6 +1,8 @@
 import { useState } from 'react'
 import styled from 'styled-components'
+import { useLocation, generatePath } from 'react-router-dom'
 import { ListItemType } from 'src/components/List'
```

Comment:
We follow a style whereby we import externally, then include a line break and import internally.